suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -684,3 +684,8 @@ export const selectPersistedDevicesStates = (state: DeviceRootState) => {
     const devices = selectDevices(state);
     return [...devices.map(d => d.id), PORTFOLIO_TRACKER_DEVICE_STATE];
 };
+
+export const selectIsNoPhysicalDeviceConnected = (state: DeviceRootState) => {
+    const devices = selectDevices(state);
+    return devices.length === 1 && devices[0].id === PORTFOLIO_TRACKER_DEVICE_ID;
```

Comment:
nit: could be simplified to just `devices?.[0].id ===  PORTFOLIO_TRACKER_DEVICE_ID`

Reply:
I think the ID on the index 0 is alway the Portfolio tracker, because it is always created already during the app init. So IMO if I would remove the check for the `devices.length`, it would not work as expected.